app/lib/three_scale/semantic_form_builder.rb

```diff
@@ -190,6 +190,21 @@ def cancel_link(path_or_label, path = nil)
       template.content_tag(:li, template.link_to(label, path), :class => 'link')
     end
 
+    def backend_api_system_name
```

Comment:
I do not think we should use that for just one form.
You can do exactly the same in the html instead of hooking to Formtastic